docs/quickstart_guide/index.rst

```diff
@@ -0,0 +1,245 @@
+.. _quickstart_guide:
+
+Quickstart Guide
+################
+
+This section aims to get you started with Lewis as quickly as possible. It is meant as a basic starting point for becoming familiar with Lewis, and to give you a broad overview of what it can do. As such, many features are skimmed over or skipped entirely. See the detailed documentation sections for a more complete overview of features.
+
+This guide is presented as a step-by-step tutorial, so skipping sections may mean you will miss steps that are required for the examples to work.
+
+
+Install Lewis
+=============
+
+The recommended way to install Lewis is via PyPI and using a virtual environment. This guide assumes you have Python and Pip installed and in your PATH.
+
+Install ``virtualenv`` and create a virtual environment. On Windows, the activate script will be located elsewhere and can be executed directly, without ``source``. This step is optional and may be skipped:
+
+::
+
+    $ pip install virtualenv
+    $ virtualenv myenv
+    $ source myenv/bin/activate
+
+Lewis can be installed with Pip using a single command:
+
+::
+
+    $ pip install lewis
+    $ lewis --version
+
+
```

Comment:
How about mentioning `$ lewis -h` to get lewis help options?

Reply:
Great idea!